src/js/utils/parser.ts

```diff
@@ -26,7 +26,7 @@ export function getAbsolutePath(path: string, base?: string): string {
         basepath = basepath.substring(protocol.length + domain.length + 1, basepath.lastIndexOf('/'));
         patharray = basepath.split('/').concat(path.split('/'));
     }
-    const result = [];
+    const result: string[] = [];
     for (let i = 0; i < patharray.length; i++) {
```

Comment:
Interesting that these failed when removing noImplicitAny

Reply:
yeah right? not quite clear on that - makes me feel more secure in removing it given it seems like there's some phantom behavior or something

Reply:
definitely agree